pantheon/src/main/java/tech/pegasys/pantheon/RunnerBuilder.java

```diff
@@ -338,6 +352,29 @@ public Runner build() {
                   vertx, dataDir, jsonRpcConfiguration, metricsSystem, jsonRpcMethods));
     }
 
+    Optional<GraphQLRpcHttpService> graphQLRpcHttpService = Optional.empty();
+    if (graphQLRpcConfiguration.isEnabled()) {
+      final GraphQLDataFetchers fetchers = new GraphQLDataFetchers(supportedCapabilities);
+      final GraphQLDataFetcherContext dataFetcherContext =
+          new GraphQLDataFetcherContext(
+              context.getBlockchain(),
+              context.getWorldStateArchive(),
+              protocolSchedule,
+              transactionPool,
+              miningCoordinator,
+              synchronizer);
+      GraphQL graphQL = null;
+      try {
+        graphQL = GraphQLProvider.buildGraphQL(fetchers);
+      } catch (final IOException ignored) {
```

Comment:
We shouldn't blindly ignore this exception or we'll wind up with an inexplicable `NullPointerException` further down the line.  Most likely should just rethrow as an unchecked exception as our startup has failed if we get an IOException at this point.

Push the catch & rethrow inside `GraphQLProvider.buildGraphQL` would likely make sense too since it knows the IOException only came while attempting to load resources that should have been in the jar.

Reply:
done